core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1911,7 +1911,6 @@ public View getPrimaryView() {
      }
 
     public void setPrimaryView(@Nonnull View v) {
-        checkPermission(Jenkins.ADMINISTER);
```

Comment:
I did put MANAGE there at the beginning of our work, I changed to ADMINISTER because I didn't remember the why. Are you sure we should remove all checkPermission and not stay to MANAGE ?

Reply:
There were some concerns about allowing this, because a user with MANAGE, if doesn't have any other permission, won't be able to set as primary view the views created by the admin, for example, because he is not allowed to see them. The use case is then a bit odd.

Also, I'm not sure which would be the outcome of the following scenario: if the admin set as primary view  a view that MANAGE can not see, and the MANAGE then go to the form and save it, if he is allowed to modify the primary view, will he be changing the primary view that the Admin saved? because there is no way that the <select> shown to the MANAGE in the form is showing that view, so it might be showing some other, and when saving, it would change...